service/controller/resource/endpoints/desired.go

```diff
@@ -26,12 +26,12 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 			Name:      "master",
 			Namespace: key.ClusterID(cr),
 			Labels: map[string]string{
-				"app":                        "master",
-				"cluster":                    key.ClusterID(cr),
-				"customer":                   key.ClusterCustomer(cr),
-				"giantswarm.io/cluster":      key.ClusterID(cr),
-				"giantswarm.io/organization": key.ClusterCustomer(cr),
-				"giantswarm.io/managed-by":   "azure-operator",
+				key.LabelApp:           "master",
+				key.LabelCluster:       key.ClusterID(cr),
+				key.LabelCustomer:      key.ClusterCustomer(cr),
+				key.LegacyLabelCluster: key.ClusterID(cr),
+				key.LabelManagedBy:     "azure-operator",
```

Comment:
(if you apply that you'll also have to sort the import)

```suggestion
				key.LabelManagedBy:     project.Name(),
```

Reply:
Applied here and other several places as well.